test/samples/mock-data.ts

```diff
@@ -16,7 +16,7 @@ const data: UserMessages = {
       "id": "draft-0",
       "threadId": "16eec6ebc087faa7",
       "labelIds": ["DRAFT"],
-      "snippet": "[cryptup:link:draft_compose:r304765387393056602] -----BEGIN PGP MESSAGE----- Version: FlowCrypt 7.3.6 Gmail Encryption Comment: Seamlessly send and receive encrypted email wcFMA0taL/zmLZUBAQ//",
+      "snippet": "[flowcrypt:link:draft_compose:r304765387393056602] -----BEGIN PGP MESSAGE----- Version: FlowCrypt 7.3.6 Gmail Encryption Comment: Seamlessly send and receive encrypted email wcFMA0taL/zmLZUBAQ//",
```

Comment:
I would actually leave some of these with "cryptup" still - to make sure both versions work (in case you haven't).